src/javascript/register.jsx

```diff
@@ -56,6 +56,7 @@ if (contextJsParameters && contextJsParameters.config && contextJsParameters.con
             submitOperation: EditPanelConstants.submitOperation.UNPUBLISH
         });
 
+        // TODO: use this action in picker actions.
```

Comment:
Alors deux choses :

- <3 si ça marche car je ne l'avais pas vu
- dans tout les cas TODO a supprimer car non lié à un ticket (pas besoin de mettre le todo car je suis en train de le faire)

Mais gros <3

Reply:
Okay, I'll take it off, I'll just add it so I don't forget it.